.gitignore

```diff
@@ -35,3 +35,5 @@ config.json
 npm-debug.log
 
 manifest.yml
+
+.idea
```

Comment:
Boro na to afereso apla o IDE to evale gia na kani ignore ta project files aftomata.